Source/WebCore/page/scrolling/AsyncScrollingCoordinator.cpp

```diff
@@ -1102,7 +1112,11 @@ bool AsyncScrollingCoordinator::isScrollSnapInProgress(ScrollingNodeID nodeID) c
 
 void AsyncScrollingCoordinator::updateScrollSnapPropertiesWithFrameView(const FrameView& frameView)
 {
-    if (auto node = downcast<ScrollingStateFrameScrollingNode>(m_scrollingStateTree->stateNodeForID(frameView.scrollingNodeID()))) {
+    auto scrollingNode = m_scrollingStateTree->stateNodeForID(frameView.scrollingNodeID());
```

Comment:
This could dynamicDowncast<>.

Reply:
The same issue. The code doesn't compile if I try to call dynamicDowncast there.